### ISSUE TITLE
[BUG] Clicking the audio play button results in a pause, rather than a replay

### ISSUE THREAD

**Issue by irfankort** [ankidroid/Anki-Android#14221]
### Checked for duplicates?

- [X] This issue is not a duplicate

### What are the steps to reproduce this bug?

The bug occurred following the notice regarding a migration process following the release of version 2.16. I made an effort to rectify it by installing an earlier version, from the GitHub archives. While this at first appeared to work, the next day, the same migration notice appeared and the bad behavior recommenced.

### Expected behaviour

With audio cards, there has always been a "play" button. Usually, clicking on the play button results in the audio file playing from the beginning -- I can click it even in the middle of playing, and it recommences from the beginning. 

### Actual behaviour

Now, the button has a grey circle around it. When I click on it, it results in the audio STOPPING. If I click on it again, it recommences from where I paused it. It makes the cards unusable. It only occurs with Ankidroid -- Anki on my PC is behaving as normal. I've search through all the settings and can't find anything. 

### Debug info

```text
AnkiDroid Version = 2.16.2

Android Version = 13

ProductFlavor = play

Manufacturer = samsung

Model = SM-G998B

Hardware = exynos2100

Webview User Agent = Mozilla/5.0 (Linux; Android 13; SM-G998B Build/TP1A.220624.014; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/115.0.5790.166 Mobile Safari/537.36

ACRA UUID = 201e8f12-31a7-40b0-9d4c-2f36789acc47

New schema = false

Scheduler = std2

Crash Reports Enabled = true

DatabaseV2 Enabled = true
```

### (Optional) Anything else you want to share?

I first raised this issue on the Ankdroid Google group:

https://groups.google.com/g/anki-android/c/WxchecaB9Zs

An experienced member of the group stated:

> When I click on it, it results in the audio STOPPING. If I click on it again, it recommences from where I paused it. It makes the cards unusable. It only occurs with Ankidroid -- Anki on my PC is behaving as normal. 

Reproduced. I think it is a (perhaps unreported) bug, not something that can be changed in the settings.

You might want to create an issue topic on the developer site: 
https://github.com/ankidroid/Anki-Android/issues/new?labels=Needs+Triage&projects=&template=bug_report_form.yml

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by Sbalal** [comment]
I am having the exact same issue. Ankidroid does not play audio files properly and before you could just press the play button even midway through and it would be fine to start from beginning. Now it freezes. Would appreciate a fix, I've previously donated to the app too as I appreciate what you guys do. Thanks

**Comment by criticalAY** [comment]
The issue is likely to be fixed when we take care of the UI changes, I raised this issue earlier regarding adding a timeline when playing audio.

**Comment by irfankort** [comment]
> The issue is likely to be fixed when we take care of the UI changes, I raised this issue earlier regarding adding a timeline when playing audio.

Thanks for your response, and for your ongoing work on this excellent open source tool. My apologies if I'm being obtuse, but what are these "UI changes"? Thanks!

**Comment by criticalAY** [comment]
The current UI of AnkiDriod I mean

**Comment by irfankort** [comment]
> The issue is likely to be fixed when we take care of the UI changes, I raised this issue earlier regarding adding a timeline when playing audio.

Ah, I saw an update to AnkiDroid and installed it with great excitement, hoping that the issue had been resolved. Nope, it's still ongoing. For now, I'm continuing to use the desktop app when I can.

**Comment by david-allison** [comment]
Related: 
* #6774
* #8781

**Comment by ODAWGG** [comment]
I am confused is this a bug or this meant to be this feature : https://github.com/ankidroid/Anki-Android/pull/8781 ?

Personally, I would love to be able to pause and play audio.

**Comment by snowtimeglass** [comment]
If I understand correctly, #8781 was merged once, but it has been changed by #14296 after #14221 discussion. As mentioned [here](https://github.com/ankidroid/Anki-Android/issues/13125#issuecomment-1704444313), the current behavior (replay) is consistent with Anki Desktop and AnkiMobile.

The current issue is that AnkiDroid doesn't have the "Pause audio" option, unlike Anki Desktop and AnkiMobile.

**Comment by irfankort** [comment]
Just to confirm that the issue has been resolved in the latest update, everything working as in the past. I guess it COULD be a feature rather than a bug, depends on your personal usage. For me, it was extremely annoying. As an OPTION, I guess it could be fine, if there is demand for it.
